mne_connectivity/__init__.py

```diff
@@ -6,9 +6,15 @@
 #
 # License: BSD (3-clause)
 
-__version__ = '0.1'
+__version__ = '0.2dev'
```

Comment:
Not the right way to format this

https://www.python.org/dev/peps/pep-0440/

```suggestion
__version__ = '0.2.dev0'
```